src/Benchmarks/InvokeBenchmarks/InvokeBenchmarks.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Need to decide if this warrants another src-level folder. Personally what I'd say is:

- src
    - Lab
        - Experiments (everything in Lab atm is moved in there)
        - Benchmarks

Benchmarks aren't really a feature of Silk.NET, which is why I don't think it warrants its own src-level folder.

Reply:
Done